x-pack/test/reporting/configs/functional.js

```diff
@@ -21,9 +21,15 @@ export async function getFunctionalConfig({ readConfigFile }) {
     junit: {
       reportName: 'X-Pack Reporting Functional Tests',
     },
-    kbnTestServer: xPackFunctionalTestsConfig.get('kbnTestServer'),
+    kbnTestServer: {
+      ...xPackFunctionalTestsConfig.get('kbnTestServer'),
+      serverArgs: [
+        ...xPackFunctionalTestsConfig.get('kbnTestServer.serverArgs'),
+        '--logging.events.log', JSON.stringify(['info', 'warning', 'error', 'fatal', 'optimize', 'reporting'])
+      ],
+    },
```

Comment:
These changes won't really take effect with this PR, since there still are no functional tests enabled. But I think it's better to make this change in this PR because it's similar to another change, and we can check for consistency